src/openms/source/ANALYSIS/OPENSWATH/OpenSwathWorkflow.cpp

```diff
@@ -908,8 +913,8 @@ namespace OpenMS
       // Then correct the start/end positions and add the extra_rt_extract parameter
       prepare_coordinates_sub(chrom_list, coordinates, transition_exp_used, 0.0, ms1);
       for (std::vector< ChromatogramExtractor::ExtractionCoordinates >::iterator it = coordinates.begin(); it != coordinates.end(); ++it)
-      {
-        it->rt_start = trafo_inverse.apply(it->rt_start) - (cp.rt_extraction_window + cp.extra_rt_extract)/ 2.0;
+      {// iterator it parse through complete vector of coordinates and all transitions have RT end/start set to +/-300 seconds.
```

Comment:
probably we dont want to write explicit numbers in here as 300 seconds is just an example value, not a fixed value